### PR TITLE
feat(cli): add app bridge version to build

### PR DESCRIPTION
**Pull request by ragi96**
Adds the appBridge version to the window. 

`serve`: `window.DevCustomBlock.dependencies.appBridge`

`deploy`: `window.<appId>.dependencies['@frontify/app-bridge']`

[Task](https://app.clickup.com/t/869309vxt)

### CODE REVIEW

File: packages/cli/src/commands/serve.ts [hunk 1]
```diff
@@ -56,6 +57,7 @@ class DevelopmentServer {
                 ],
                 define: {
                     'process.env.NODE_ENV': JSON.stringify('development'),
+                    'DevCustomBlock.packages.appBridge': getAppBridgeVersion(process.cwd()),
```
Comment (ragi96) [inline packages/cli/src/commands/serve.ts:60]: @SamuelAlev not sure if we should adjust it to `DevCustomBlock.<port>.packages.appBridge`, so you could serve multiple blocks with different appBridge versions

File: packages/cli/src/utils/appBridgeVersion.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { join } from 'node:path';
+import { reactiveJson } from './reactiveJson.js';
+import { PackageJson } from './npm.js';
+
+export const getAppBridgeVersion = (rootPath: string) => {
+    const packageJson = reactiveJson<PackageJson>(join(rootPath, 'package.json'));
+    return getMajorVersion(packageJson.dependencies['@frontify/app-bridge']);
```
[the remark this answers is not in the document]
Comment (ragi96) [reply packages/cli/src/utils/appBridgeVersion.ts]: Was thinking about that too, so its now adding the full version to the window 👍

File: packages/cli/src/utils/appBridgeVersion.ts
Comment (SamuelAlev) [inline packages/cli/src/utils/appBridgeVersion.ts:1]: Could you add a test for the utility?

File: .changeset/warm-apples-press.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/frontify-cli": minor
+---
+
+feat: adds the appBridge version to the window
```
Comment (SamuelAlev) [inline .changeset/warm-apples-press.md:5]:
```suggestion
feat: adds the appBridge version to the block bundle
```

File: packages/cli/src/commands/serve.ts [hunk 4]
```diff
@@ -56,6 +57,7 @@ class DevelopmentServer {
                 ],
                 define: {
                     'process.env.NODE_ENV': JSON.stringify('development'),
+                    'DevCustomBlock.packages.appBridge': JSON.stringify(getAppBridgeVersion(process.cwd())),
```
[the remark this answers is not in the document]
Comment (ragi96) [reply packages/cli/src/commands/serve.ts]: Yes, (we just need to adjust the `localblockdevelopment.ts` a bit to move the initialization of `this.appBridge` after the import to have it available)

Or what's the question?

[the remark this answers is not in the document]
Comment (SamuelAlev) [reply packages/cli/src/commands/serve.ts]: This line will replace `DevCustomBlock.dependencies.appBridge` in the code with the version, however I don't see `DevCustomBlock.dependencies.appBridge` anywhere.

[the remark this answers is not in the document]
Comment (ragi96) [reply packages/cli/src/commands/serve.ts]: For me its working perfectly fine:
Screenshot 2023-11-06 at 18 31 56

[the remark this answers is not in the document]
Comment (SamuelAlev) [reply packages/cli/src/commands/serve.ts]: Nevermind I meant something else but checked a bit the produced code and it is fine

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**